centaur/src/main/resources/standardTestCases/local_bourne/local_bourne.wdl

```diff
@@ -0,0 +1,20 @@
+task shell {
+  command {
+    # The SHELL variable isn't set in /bin/sh on alpine, busybox or ubuntu so there is this abomination instead.
+    ps auxwww | head -n 2 | tail -n 1 | cut -c23-50
```

Comment:
@kshakir I expanded the original code a bit to be able to take `job-shell` from the backend config, and with some Centaur whispering I got an alpine/`/bin/sh` test running. The one bit that's still irksome is the "what shell is this" logic since `$SHELL` isn't being set on `/bin/sh`. Suggestions appreciated. 🙂

Reply:
Woo!! Nice to see WDL getting some bourne love. ❤️ 

I think your command is overkill though. The images are intentionally tiny and provide very little functionality beyond the POSIX spec, so they won't have your `SHELL` variable etc. But because the busybox images (including alpine) don't have bash, the fact that this runs at all is amazing. A simple `echo hello world` would suffice.

If you really want to push the limits of stress testing our POSIX support... add another call to the same wdl with [`busybox:1.28.3`](https://hub.docker.com/r/library/busybox/tags/) that's half the size of [`alpine:3.6`](https://hub.docker.com/r/library/alpine/tags/)!